starter-code/scripts/models/article.js

```diff
@@ -20,10 +20,18 @@
 
   // Set up a DB table for articles.
   Article.createTable = function() {
+    var inputString = 'CREATE TABLE IF NOT EXISTS articles_table (id INTEGER PRIMARY KEY, title VARCHAR, category VARCHAR, author VARCHAR, authorUrl VARCHAR, publishedOn DATE, body TEXT);';
```

Comment:
While valid, `TEXT` as a data type is deprecated. Moving forward you should use VARCHAR